ex12.js

```diff
@@ -0,0 +1,10 @@
+const getRandomDigit = getRandomIntegerGenerator(0, 9);
+console.log(getRandomDigit); // random number between 0 and 9
+
+console.log(getRandomIntegerGenerator(-10, 10)); // random number between -10 and 10
+
+function getRandomIntegerGenerator(min, max){
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min))
```

Comment:
please test it, it is not working as expected :)

Reply:
could you explain whats the problem? it works fine for me. the only thing i can think of is that the boundries should be included in the generator, is that it?